src/content/validation.md

```diff
@@ -70,46 +70,45 @@ After selecting, we can use `isFormValid$` with the async pipe in our template
 Much like we would create validators, we can create specific rules per-field
 
 ```ts
-export const isAgeValidSelector = createSelector(
+const humanAgeValid = isBetweenNumber(14, 40);
+const elfAgeValid = isBetweenNumber(80, 800);
+const tieflingAgeValid = isBetweenNumber(35, 53);
```

Comment:
I thought that we could just avoid explaining it, I can just say it will return true if the value is between range